test/utils/images/images.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package images
+
+import "os"
+
+func GetPauseImage() string {
+	if pullSpec, ok := os.LookupEnv("E2E_PAUSE_IMAGE_URL"); ok {
+		return pullSpec
+	}
+	return PauseImage
```

Comment:
...and fix the name here and in all the relevant places as reported by go vet/lint.
Up to you, I'm fine both ways.